common/icons/components/ClosedFolder.tsx

```diff
@@ -0,0 +1,51 @@
+// This is autogenerated from an SVG file
+// Do not edit this file directly; edit the SVG file and run "yarn run-svgr"
+import { IconSvg } from '../types';
+
+const SvgClosedFolder: IconSvg = props => (
+  <svg
+    width={24}
+    height={24}
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <mask
+      id="mask0_2291_633"
+      style={{
+        maskType: 'alpha',
+      }}
+      maskUnits="userSpaceOnUse"
+      x={0}
+      y={0}
+      width={24}
+      height={24}
+    >
+      <rect y={0.5} width={24} height={24} fill="#D9D9D9" />
```

Comment:
Should fill values be `currentColor` and I think it's meant to have a `icon__shape` class somewhere as well (looking at our other icons).